src/lib/dnssd/Resolver_ImplMinimalMdns.cpp

```diff
@@ -281,6 +281,7 @@ class MinMdnsResolver : public Resolver, public MdnsPacketDelegate
     CHIP_ERROR ResolveNodeId(const PeerId & peerId, Inet::IPAddressType type) override;
     CHIP_ERROR DiscoverCommissionableNodes(DiscoveryFilter filter = DiscoveryFilter()) override;
     CHIP_ERROR DiscoverCommissioners(DiscoveryFilter filter = DiscoveryFilter()) override;
+    CHIP_ERROR StopDiscovery() override { return CHIP_ERROR_NOT_IMPLEMENTED; }
```

Comment:
could we clear out mActiveResolves here if they are looking for a particular node? 
Or is that not the intent?

Reply:
This API is meant to "stop" whatever DiscoverCommissionableNodes/DiscoverCommissioners start.  mActiveResolves may be related to those, or it may not be....

Reply:
Active resolves have a type of resolve which can be Browse:

https://github.com/project-chip/connectedhomeip/blob/master/src/lib/dnssd/ActiveResolveAttempts.h#L180

It seems to me like stop discovery is supposed to stop any callback being triggered if it was started by a browse. This includes any retransmit attempts if no replies were ever received (not sure if minmdns does retry send on browse, but in case we do support it now or in the future)

Reply:
> It seems to me like stop discovery is supposed to stop any callback being triggered if it was started by a browse.

Fwiw, the caller does not rely on that.  What it really needs to do is clean up anything allocated by the `Discover*` calls.

But yes, cleaning out the `IsBrowse()` things from `mActiveResolves` would be fine.  

We don't seem to have an API for this sort of thing right now.  Do you just want me to add a `CompleteAllBrowses` on `ActiveResolveAttempts`?  Or something else?

Reply:
Something like that. I believe other system dnssd servers on stop would release resources allocated and stop re-issuing queries.

In MinMDNS land, the resources allocated is the mActiveResolves list (and it will use that to re-issue queries). May as well clean it up if we are asked to stop.

Reply:
Done.